recipes/strawberryperl/all/test_package/conanfile.py

```diff
@@ -14,7 +12,6 @@ def build_requirements(self):
         self.tool_requires(self.tested_reference_str)
 
     def test(self):
-        if can_run(self):
-            self.run("perl --version")
-            perl_script = os.path.join(self.source_folder, "list_files.pl")
-            self.run(f"perl {perl_script}", env="conanbuild")
+        self.run("perl --version")
```

Comment:
Can you please add a test for conf info? trying to get an example for the requests in #13490 

the docs say this should work in both majors 🤞 

```suggestion
        self.run("perl --version")
        conf_perl = self.conf_info.get("user.strawberryperl:perl")
        self.run(f"{conf_perl} --version")
```

Reply:
it's broken in test package for conan < 1.53.0